utils/net_utils/Cargo.toml

```diff
@@ -0,0 +1,14 @@
+[package]
+name = "net_utils"
+version = "0.1.0"
+authors = ["ChainSafe Systems <info@chainsafe.io>"]
+edition = "2018"
+
+[dependencies]
+indicatif = "0.15.0"
+futures = "0.3.5"
+async-std = { version = "1.6.3" }
+isahc = "0.9.11"
```

Comment:
`isahc` was selected because

1. `reqwest` depends on `tokio`, not `async-std`
2. I attempted `surf` but got some issues https://github.com/http-rs/surf/issues/218#issuecomment-721774449
3. `surf` uses `isahc` as one of the backends anyways.